packages/config/src/handlers/dashboardQueries/defaultQueries.ts

```diff
@@ -100,7 +100,7 @@ const registrations = ({ lastUpdatedAt }: { lastUpdatedAt: string }) => ({
     },
     {
       $addFields: {
-        allTasks: { $concatArrays: ['$task', '$task_history'] }
+        allTasks: { $concatArrays: ['$task_history', '$task'] }
```

Comment:
@Zangetsu101 just out of interest, what was the reason these need to be flipped?

Reply:
This was from a previous change, the queries in core just hadn't been updated. If I am not wrong then it had something to do with the order of the `allTasks`. Maybe there is an assumption down the line that the tasks are sorted in ascending order.

Reply:
btw @rikukissa the fix for this bug is actually here: https://github.com/opencrvs/opencrvs-countryconfig/pull/399